apps/i18n/aichat/en_us.json

```diff
@@ -1,7 +1,21 @@
 {
+  "aichatWorkspaceDescription": "The area where students can interact directly with the chatbot they've created.",
   "aichatWorkspaceHeader": "AI Chat",
+  "chatComponentsHeader": "AI Chat Components",
+  "colonSpace": ": ",
   "continue": "Continue",
+  "instructions": "Instructions",
+  "instructionsDescription": "Clear directions and goals for each level.",
+  "learnMoreHereInBrackets": "[Learn more here.]",
```

Comment:
Since the link is at the end of the paragraph, how you implemented this formatting works pretty well, avoiding using <SafeMarkdown> to render the link that will require additional styling.